papaya/build.gradle

```diff
@@ -1,9 +1,13 @@
 apply from: "../gradle/shared.gradle"
 
+sourceCompatibility = 1.6
+targetCompatibility = 1.6
+ideaSetModuleLevel(idea, targetCompatibility)
```

Comment:
This would likely be better in a subProjects block in the root build.gradle file - avoid people having to remember to do it every time.